build_library/vm_image_util.sh

```diff
@@ -945,9 +964,17 @@ EOF
 }
 
 _write_ova_bundle() {
-    vmdk-convert ${VM_DST_IMG} ${VM_TMP_DIR}/vm.vmdk
```

Comment:
Use `....DISK_FORMAT=vmdk_stream` and then $VM_DST_IMG is already in the correct format and ensures that the ovf+vmdk is equivalent to the ova.